jenkins-master/jobs/scripts/workspace/submission.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import re
+import socket
+import time
+from urlparse import urljoin, urlparse
+import uuid
+
+from config import config
+
+here = os.path.dirname(os.path.abspath(__file__))
+
+LOOKUP_FRAGMENT = 'api/project/{repository}/resultset/?revision={revision}'
+REVISON_FRAGMENT = '/#/jobs?repo={repository}&revision={revision}'
```

Comment:
[Nit] The names `LOOKUP_FRAGMENT` and `REVISION_FRAGMENT` do not distinguish the two strings and it's not clear what they are intended for. Maybe `RESULTSET_FRAGMENT` and `JOB_FRAGMENT` would be clearer.